feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt

```diff
@@ -45,8 +77,20 @@ fun ApartmentContainer(navigateToRepair : () -> Unit) {
     ) {
         Column {
             TitleApartmentScreen()
-            PaymentsSection(Modifier, "Junary", "990.00", "Jan 23 2022")
-            RepairsSection { navigateToRepair.invoke() }
+            userData.paymentDetails?.currentMonth?.let {
+                userData.paymentDetails.deadline?.let { it1 ->
+                    PaymentsSection(
+                        modifier = Modifier,
+                        month = it,
+                        quantityToPay = userData.paymentDetails.rentDebt.toString(),
+                        limitDateToPay = it1
+                    )
+                }
+            }
+            RepairsSection(
+                navigateToRepair ={ navigateToRepair.invoke() },
```

Comment:
you can change navigateToRepair.invoke() to navigateToRepair()

Reply:
Comment resolved